app/controllers/v0/sessions_controller.rb

```diff
@@ -13,8 +13,24 @@ def show
     end
 
     def destroy
-      @session.destroy
-      head :no_content
+      logout_request = OneLogin::RubySaml::Logoutrequest.new
+      logger.info "New SP SLO for userid '#{@session.uuid}'"
+
+      if saml_settings.name_identifier_value.nil?
+        saml_settings.name_identifier_value = @session.uuid
+      end
+
+      saml_settings.security[:logout_requests_signed] = true
+      saml_settings.security[:embed_sign] = true
+
+      render json: { logout_via_get: logout_request.create(saml_settings, RelayState: @session.token) }, status: 202
+    end
+
+    def saml_logout
```

Comment:
Disclaimer: nitpicky, you may choose ignore. 

Perhaps a rename to `saml_logout_callback` is more informative?